pinc/unicode.inc

```diff
@@ -310,3 +310,68 @@ function guess_string_encoding($text)
     # Give up and return ISO-8859-1
     return 'ISO-8859-1';
 }
+
+# Return a list of Unicode codepoints we want to replace with some ASCII
+# equivalence. We convert these codepoints to ASCII up on page save in
+# DPage.inc.
+function get_utf8_to_ascii_codepoints()
+{
+    return [
```

Comment:
The first three replacement characters below are delimited with apostrophes; the rest of them with  double quotes. All of the codepoints are delimited by apostrophes. Just wondering whether it should be all one or the other?

Reply:
It's mostly irrelevant. If we used all of one we'd need to escape the character in one of the strings.